### ISSUE TITLE
error when some pdb files cannot have graph generated

### ISSUE THREAD

**Issue by LilySnow** [DeepRank/iScore#16]
When some pdb files cannot have graph generated, we will have the error message below.  In this situation, all the pdb files have haddock energy (`Energy.dat`) and some models are missing from graphrank file (`GraphRank.dat`). 

> Traceback (most recent call last):
>   File "/home/lixue/tools/iScore/bin/iScore.predict", line 63, i
>     iscore()
>   File "/home/lixue/tools/iScore/iScore/function/score.py", line
>     self.read_graphrank()
>   File "/home/lixue/tools/iScore/iScore/function/score.py", line
>     data.append(self.features[m]['grank'])
> KeyError: 'grank'

This bug can be fixed by switching the order of the following two lines (quick and dirty): 

From 
```
          self.read_graphrank()
         self.read_energy()
```
to 

```         
          self.read_energy()
          self.read_graphrank()
```

**Comment by CunliangGeng** [comment]
if some models did not generate graphs, there should be a warning like `Warning : Graph ** not exported (num_nodes = 0)' **)`. Did you see the warning ?

**Comment by LilySnow** [comment]
Yes. The warnings are there.

**Comment by CunliangGeng** [comment]
I have fixed the bug, please review PR #17.
